lib/capybara/webkit/driver.rb

```diff
@@ -415,11 +416,26 @@ def deprecate_and_replace_with_config(deprecated_method, config_syntax = depreca
     def encode_args(args)
       args.map do |arg|
         if arg.is_a?(Capybara::Webkit::Node)
-          { ELEMENT: arg.native }.to_json
+          { 'ELEMENT' => arg.native }.to_json
         else
           arg.to_json
         end
       end
     end
+
+    def decode_result(result)
+      case result
```

Comment:
@jferris  I think that becomes unwieldy when talking about structures containing structures that may contain elements, etc.  How about just doing what the webdriver spec does and use a more complicated key instead of 'ELEMENT'  ("element-6066-11e4-a52e-4f735466cecf" in the webdriver spec) - Probably shouldn't use the same key since the webdriver spec requires the element identifiers used as the values to be UUID - but it would reduce the chance of collision while not doubling the number of objects.